src/python/WMCore/ReqMgr/CherryPyThreads/StatusChangeTasks.py

```diff
@@ -6,9 +6,76 @@
 
 from Utils.CherryPyPeriodicTask import CherryPyPeriodicTask
 from WMCore.Services.WMStats.WMStatsReader import WMStatsReader
+from WMCore.Services.WorkQueue.WorkQueue import WorkQueue
 from WMCore.Services.RequestDB.RequestDBWriter import RequestDBWriter
 from WMCore.Services.WMStats.DataStruct.RequestInfoCollection import RequestInfoCollection
 
+
+def moveForwardStatus(reqDBWriter, requestStatus):
```

Comment:
I think a doc string would be welcomed here. As a recall that `requestStatus` contains workflow names and status *based* on the workqueue elements.